### ISSUE TITLE
Regression tests on GH failing to install h5py

### ISSUE THREAD

**Issue by orso82** [gafusion/omas#175]
https://github.com/gafusion/omas/runs/4088577606?check_suite_focus=true

**Comment by orso82** [comment]
@kalling do you have any insights as to why this might be the case?

**Comment by kalling** [comment]
@orso82 Is it possible the dependency isn't yet available for Python 3.10?

**Comment by kalling** [comment]
@orso82 Looking more closely at the logfile that probably isn't it. My guess is a symlink needs to be created for the "missing" library.

**Comment by orso82** [comment]
Ah, I did not realize the tests were run in Python 3.10. It's possible that the package for h5py have not been generated for that Python version, or something like that?

**Comment by kalling** [comment]
yeah.  Also I see " setup.py:67: RuntimeWarning: NumPy 1.19.3 may not yet support Python 3.10."  So maybe my initial impression was correct after all.

**Comment by orso82** [comment]
I have just pushed a commit that fixes the version of Python for the regression tests to be 3.6 https://github.com/gafusion/omas/commit/f45902a821539e2a62fa74a214108786de8148fb

**Comment by kalling** [comment]
I would do 3.7 since it is our primary, but I would imagine that would also work.

**Comment by kalling** [comment]
<details>
<summary>I guess there are a lot not available for 3.10 yet.</summary>

```
    blis_info:
      libraries blis not found in ['/opt/hostedtoolcache/Python/3.10.0/x64/lib', '/usr/local/lib', '/usr/lib64', '/usr/lib', '/usr/lib/x86_64-linux-gnu']
      NOT AVAILABLE
  
    openblas_info:
      libraries openblas not found in ['/opt/hostedtoolcache/Python/3.10.0/x64/lib', '/usr/local/lib', '/usr/lib64', '/usr/lib', '/usr/lib/x86_64-linux-gnu']
      NOT AVAILABLE
  
    atlas_3_10_blas_threads_info:
    Setting PTATLAS=ATLAS
      libraries tatlas not found in ['/opt/hostedtoolcache/Python/3.10.0/x64/lib', '/usr/local/lib', '/usr/lib64', '/usr/lib', '/usr/lib/x86_64-linux-gnu']
      NOT AVAILABLE
  
    atlas_3_10_blas_info:
      libraries satlas not found in ['/opt/hostedtoolcache/Python/3.10.0/x64/lib', '/usr/local/lib', '/usr/lib64', '/usr/lib', '/usr/lib/x86_64-linux-gnu']
      NOT AVAILABLE
  
    atlas_blas_threads_info:
    Setting PTATLAS=ATLAS
      libraries ptf77blas,ptcblas,atlas not found in ['/opt/hostedtoolcache/Python/3.10.0/x64/lib', '/usr/local/lib', '/usr/lib64', '/usr/lib', '/usr/lib/x86_64-linux-gnu']
      NOT AVAILABLE
  
    atlas_blas_info:
      libraries f77blas,cblas,atlas not found in ['/opt/hostedtoolcache/Python/3.10.0/x64/lib', '/usr/local/lib', '/usr/lib64', '/usr/lib', '/usr/lib/x86_64-linux-gnu']
      NOT AVAILABLE
  
    accelerate_info:
      NOT AVAILABLE
```
</details>

**Comment by orso82** [comment]
Yay! The regression tests pass with Python 3.6.
Many thanks for immediately being able to point me in the right direction :)
